PIL/Image.py

```diff
@@ -1262,6 +1335,7 @@ def getprojection(self):
         return [i8(c) for c in x], [i8(c) for c in y]
 
     def histogram(self, mask=None, extrema=None):
+        # type: (Optional[Image], Optional[Any]) -> List[int]
```

Comment:
You're referring to the extrema parameter? I've looked at this again, and added an `Extrema` alias to aliases.py; let me know what you think.